tools/refengine/root.go

```diff
@@ -46,7 +46,9 @@ func (root *MerkleRoot) UnmarshalJSON(b []byte) (err error) {
 	}
 
 	mediaTypeInterface, ok := data["mediaType"]
-	if ok {
+	if !ok {
```

Comment:
You want reformat all entries in `root *MerkleRoot`, right? If consumer use a evaluated `root` to call `root.UnmarshalJSON(b)`, it will possibly  keep the old `mediatype` after calling, right?

Reply:
> You want reformat all entries in `root *MerkleRoot1, right?

Yeah.  I want the state of `root` to depend only on the unmarshaled JSON, and not on the earlier`root` state.